ratelimit/ratelimit.go

```diff
@@ -431,3 +432,7 @@ func Headers(s *Settings, retryAfter int) http.Header {
 		RetryAfterHeader: []string{strconv.Itoa(retryAfter)},
 	}
 }
+
+func getHashedKey(clearText string) string {
+	return fmt.Sprintf("%x", sha256.Sum256([]byte(clearText)))
```

Comment:
lets use a chance and change to https://golang.org/pkg/encoding/hex/#EncodeToString

Reply:
It’s not easily possible because we get an array not a slice so Sprintf is the simpler one.

Reply:
```go
h := sha256.Sum256([]byte(clearText))
return hex.EncodeToString(h[:])
```

Reply:
The change showed in a benchmark `3->2` allocs and speedup of 2x for this little function.